src/mpid/ch4/src/ch4r_rma.h

```diff
@@ -71,6 +71,7 @@ static inline int MPIDIG_do_put(const void *origin_addr, int origin_count,
     MPIR_T_PVAR_TIMER_START(RMA, rma_amhdr_set);
     am_hdr.src_rank = win->comm_ptr->rank;
     am_hdr.target_disp = target_disp;
+    am_hdr.target_lb = 0;
```

Comment:
Maybe it would be clearer if I didn't zero initialize the lower bound here. It should not affect the outcome.